server/routes/reports/details/incidentDateAndTimeFields.ts

```diff
@@ -27,4 +27,4 @@ export const incidentDateAndTimeFieldNames = [
   hoursFieldName,
   minutesFieldName,
 ] as const satisfies (keyof typeof incidentDateAndTimeFields)[]
-export type IncidentDateAndTimeFieldNames = (typeof incidentDateAndTimeFieldNames)[number]
```

Comment:
for some reason, `IncidentDateAndTimeFieldNames` was considered `any` even though incidentDateAndTimeFieldNames had a very precise type